cypress/e2e/cart.cy.js

```diff
@@ -1,57 +1,63 @@
+import LandingPage from '../support/page-objects/Landing.page.js'
+import InventoryPage from '../support/page-objects/Inventory.page.js'
+import CartPage from '../support/page-objects/Cart.page'
+import CheckoutPage from '../support/page-objects/Checkout.page.js'
+
 describe('Swag Labs tests',()=>{
   beforeEach(()=>{
-    cy.login('standard_user','secret_sauce')
+    LandingPage.openPage();
+    LandingPage.login('standard_user','secret_sauce');
   })
     it('Error wih no data',()=>{
-        cy.get('.shopping_cart_link').click()
-        cy.get('[data-test="checkout"]').click()
-        cy.get('[data-test="continue"]').click()
-        cy.get('.error-message-container').should('have.text','Error: First Name is required');
+        InventoryPage.CartButton.click();
+        CartPage.CheckoutButton.click();
+        CheckoutPage.Continue.click();
+        CheckoutPage.ErrorPrompt.should('have.text','Error: First Name is required');
     })    
 
     it('Successful shopping flow',()=>{  
-        cy.get('#shopping_cart_container').click()
-        cy.get('.cart_quantity').should('not.exist')
-        cy.get('[data-test="continue-shopping"]').click()
-        cy.get('[data-test="add-to-cart-sauce-labs-bike-light"]').click()
-        cy.get('.shopping_cart_badge').should('be.visible') 
-        cy.get('.shopping_cart_link').click()
-        cy.get('.inventory_item_name').should('have.text','Sauce Labs Bike Light')
-        cy.get('.inventory_item_price').should('have.text','$9.99')
-        cy.get('[data-test="remove-sauce-labs-bike-light"]').should('be.visible')
-        cy.get('[data-test="continue-shopping"]').should('be.visible')
-        cy.get('[data-test="checkout"]').should('be.visible')
-        cy.get('[data-test="checkout"]').click()
-        cy.get('[data-test="firstName"]').type('katarina')
-        cy.get('[data-test="lastName"]').type('barjaktarovic')
-        cy.get('[data-test="postalCode"]').type('11090')
-        cy.get('[data-test="continue"]').click()
-        cy.get('.title').should('have.text','Checkout: Overview')
-        cy.get('.cart_item').should('contain','Sauce Labs Bike Light')
-        cy.get('[data-test="finish"]').click()
-        cy.get('.complete-header').should('have.text','THANK YOU FOR YOUR ORDER')
+        CartPage.ShopingCartContainer.click()
+        CartPage.CartQuantity.should('not.exist')
+        CartPage.ContinueShoppingButton.click()
```

Comment:
everything looks really great! it's really clear and easily readable. the only suggestion that I can add, but this is only improvement, you can combine multiple actions into one method, for example, these 3 lines can be for example verifyTheCartIsEmpty. But overall, really nicely done.